x/bsonx/bsoncore/array_test.go

```diff
@@ -299,7 +299,7 @@ func TestArray(t *testing.T) {
 					'\x00', '\x00',
 				},
 				`[[null,null]]`,
-				`Array(19)[[null ,null ]]`,
+				`Array(19)[Array(11)[null,null]]`,
```

Comment:
This was using `docsAsArray()` (now removed) but is now actually using `Array#DebugString()`.

Reply:
Ah nice, this is more intuitive output for an array of an array.